capstone/src/main/java/com/google/sps/Objects/response/ChallengeResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.Objects.response;
+
+import com.google.sps.Objects.Challenge;
+
+public class ChallengeResponse {
+
+  private Challenge challenge;
+  private boolean isCompleted;
+
+  public ChallengeResponse(Challenge challenge, boolean isCompleted) {
+    this.setChallenge(challenge);
+    this.isCompleted = isCompleted;
```

Comment:
Could you rename this, and elsewhere in the Challenge class and Challenge Servlet, to something like userHasCompleted instead? I feel isCompleted is a little vague as to its intent, and I needed to look through the code to identify what exactly the purpose of this was